### PR TITLE
use versioneer for version management

### LINKED ISSUE #87: use versioneer
Stream-line version information with https://github.com/warner/python-versioneer and maintain the version as the git tag.

### PR DESCRIPTION
**Pull request by orbeckst**
- fix #87 
- add versioneer with proper configuration for test coverage
- add a small test for the new `propka.__version__` attribute
- make docs automatically use the version
- fixed omitted author in docs

### CODE REVIEW

File: propka/__init__.py [hunk 1]
```diff
@@ -18,3 +18,7 @@
            "hybrid36", "iterative", "input", "lib", "ligand_pka_values",
            "ligand", "molecular_container", "output", "parameters",
            "protonate", "run", "vector_algebra", "version"]
+
+from ._version import get_versions
+__version__ = get_versions()['version']
+del get_versions
```
Comment (sobolevnrm) [inline propka/__init__.py:24]: Why do you need to delete `get_versions` here?

Comment (orbeckst) [reply propka/__init__.py:24]: This is versioneer’s boiler-plate code. It’s done whenever versioneer adds itself. If you want to keep `get_versions()` you can but I think generally it’s thought that one wants to keep the name space clean and you just access the version as `propka.__version__`. If you need get `get_versions()` for something specific you could use it from `propka._version.get_versions()`. 

Also, not having it exposed allows us to not having to test it explicitly and treat it as implementation detail.

Comment (sobolevnrm) [reply propka/__init__.py:24]: Thanks.  I was mainly just curious.

**Comment by sobolevnrm** [comment]
@orbeckst - this looks great.  Do you want to do the merging honors? :)

**Comment by orbeckst** [comment]
Sure, happy to :-).

**Pull request merged**